### DISCUSSION TITLE
Radio / Checkbox buttons in different containers

### DISCUSSION THREAD

**Discussion by 3lyrion** [mikke89/RmlUi#705] [Q&A] [answered]
![image](https://github.com/user-attachments/assets/25b3ccd0-91e6-42d1-ad19-d189783cbe44)
![image](https://github.com/user-attachments/assets/989e90ef-e5d2-4423-b011-2fd7a6b18868)
I can't see the value of "checked". After clicking on the button, the "checked" pseudo-class is **permanently** activated

**Comment by mikke89** [comment]
Hey. Keep in mind that there are no default styles in RmlUi. So even built-in elements such as checkboxes need to be styled.

Please see our samples and documentation. Here is one example:

https://github.com/mikke89/RmlUi/blob/7a8aceafcc501c5b09ea819bed96d273d9382749/Tests/Data/style.rcss#L123-L141

**Reply by 3lyrion** [reply]
I am creating input.radio via Lua. Am I doing something wrong?

![image](https://github.com/user-attachments/assets/d2ab43c5-fe8c-44e6-81a0-5ae588b1f6e3)

**Reply by 3lyrion** [reply, marked as answer]
These elements had to be in a form.
